tests/foreman/api/test_errata.py

```diff
@@ -288,33 +942,17 @@ def test_positive_list_updated(module_org, custom_repo, target_sat):
     updated = [errata.updated for errata in erratum_list]
     assert updated == sorted(updated)
 
-
-@pytest.mark.tier3
-def test_positive_sorted_issue_date_and_filter_by_cve(module_org, custom_repo, target_sat):
-    """Sort by issued date and filter errata by CVE
-
-    :id: a921d4c2-8d3d-4462-ba6c-fbd4b898a3f2
-
-    :Setup: Errata synced on satellite server.
-
-    :Steps: GET /katello/api/errata
-
-    :expectedresults: Errata is sorted by issued date and filtered by CVE.
-
-    :CaseLevel: System
-    """
     # Errata is sorted by issued date.
     erratum_list = target_sat.api.Errata(repository=custom_repo['repository-id']).search(
         query={'order': 'issued ASC', 'per_page': '1000'}
     )
     issued = [errata.issued for errata in erratum_list]
     assert issued == sorted(issued)
-
     # Errata is filtered by CVE
     erratum_list = target_sat.api.Errata(repository=custom_repo['repository-id']).search(
         query={'order': 'cve DESC', 'per_page': '1000'}
     )
-    # Most of Errata don't have any CVEs. Removing empty CVEs from results
+    # Most of Errata won't have any CVEs. Removing empty CVEs from results
```

Comment:
while you're at it ;)
```suggestion
    # Most Errata won't have any CVEs. Removing empty CVEs from results
```